carbonplan_tonyear/__init__.py

```diff
@@ -0,0 +1,7 @@
+import pathlib
+
+from intake import open_catalog
+
+cat_dir = pathlib.Path(__file__)
+cat_file = str(cat_dir.parent / "data/catalog.yaml")
+cat = open_catalog(cat_file)
```

Comment:
```suggestion
from .core import calculate_tonyears
```

Reply:
So, my last hurdle is that the imports in `carbonplan_tonyear/__init__.py` are raising pre-commit flake8 fail. 

> carbonplan_tonyear/__init__.py:1:1: F401 '.core.calculate_tonyears' imported but unused

How to navigate?

Reply:
Telling flake8 to ignore imports in `__init__.py` with `# flake8: noqa`.